### PR TITLE
[Mellanox] Update Nvidia sai.profile SKU files to have common file

### PR DESCRIPTION
**Pull request by noaOrMlnx**
Depends on https://github.com/sonic-net/sonic-sairedis/pull/1352

Add new common sai.profile,
Update Nvidia/Mellanox sai.profile of all SKUs to remove common parameters.

#### Why I did it
To have the ability to add common parameters to only 1 file instead of all SKUs.
##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
Add a new common file with all common key-value parameters. 
also, updated syncd_init_common.sh to take common parameters from the new file in https://github.com/sonic-net/sonic-sairedis/pull/1352

#### How to verify it
Run the script with changed .profile files, and make sure all common parameters exist in final sai.profile. 

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211
- [ ] 202305
- [x] 202311

#### Tested branch (Please provide the tested image version)

- [x] master-internal-build

#### Description for the changelog

Add new common sai.profile,
Update Nvidia/Mellanox sai.profile of all SKUs to remove common parameters.

### CODE REVIEW

**Review by nazariig — APPROVED** [review]

**Comment by nazariig** [comment]
**DEPENDS:** https://github.com/sonic-net/sonic-sairedis/pull/1352

File: platform/mellanox/docker-syncd-mlnx/sai-common.profile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+SAI_DUMP_STORE_PATH=/var/log/mellanox/sdk-dumps
+SAI_DUMP_STORE_AMOUNT=10
+SAI_DUMP_MFT_CFG_PATH=/etc/mft/fwtrace_cfg
```
Comment (dprital) [inline platform/mellanox/docker-syncd-mlnx/sai-common.profile:3]: @noaOrMlnx , I see two kind of paths. One is the one in the default profile and the other appear on several SKUs with the path of:
SAI_DUMP_MFT_CFG_PATH=/etc/sonic/mft/fwtrace_cfg

Please check if this is a mistake and all should point to the same path

**Review by dprital — COMMENTED** [review]
@noaOrMlnx , please check the comment I rasied

**Review by dprital — APPROVED** [review]

**Review by kcudnik — APPROVED** [review]

**Review by liat-grozovik — APPROVED** [review]

**Pull request merged**